packages/formdescriptioneditors/frontend/sirius-components-formdescriptioneditors/src/Group.tsx

```diff
@@ -67,7 +67,11 @@ const useGroupEntryStyles = makeStyles<Theme, GroupStyleProps>((theme) => ({
     borderColor: ({ borderStyle }) => (borderStyle ? borderStyle?.color || 'transparent' : 'gray'),
     borderStyle: ({ borderStyle }) => borderStyle?.lineStyle || 'solid',
     borderRadius: ({ borderStyle }) => (borderStyle ? borderStyle.radius : 10),
-    paddingTop: '1px',
+    margin: ({ borderStyle }) => (borderStyle ? borderStyle?.margin : null),
+    padding: ({ borderStyle }) => (borderStyle ? borderStyle?.padding : null),
+    gap: ({ borderStyle }) => (borderStyle ? borderStyle?.gap : null),
```

Comment:
The gap should be between the widgets, which are not the direct chdilren of the group.

Reply:
Done